planet/src/main/webapp/itinerary.html

```diff
@@ -87,4 +87,23 @@
             </div>
         </div>
     </body>
+    <script>
+        function initAutocomplete() {
+            let startAddress = document.getElementById('starting-address');
+            let eventAddress = document.getElementById('add-event-address');
+
+            let options = {
+                types: ['geocode']
+            };
+            autocomplete = new google.maps.places.Autocomplete(startAddress,options); 
```

Comment:
do you need to do any setFields or addListeners on this object?

Reply:
Yes, in line 100 setFields() is used to specify which data to return, which in this case is the address component. Line 101 adds a listener to update the input box when a new search is performed

Reply:
you're creating the autocomplete object, and them immediately replacing it with a different one. If you mean to setsField and addListener on both of them, you'll need to change this code to use a different variable name.